guides/common/modules/proc_configuring-pull-transport.adoc

```diff
@@ -0,0 +1,31 @@
+[id="configuring-pull-provider_{context}"]
+= Configuring Remote Execution for Pull Client
+
+By default, Remote Execution uses SSH as the transport mechanism for the Script provider.
+However, Remote Execution also offers a pull-based transport option, which communicates with hosts via MQTT.
+This is a good option if you are migrating from `katello-agent` which also uses pull-based communication.
+
+If you wish to use Remote Execution with the pull-based transport, run the following `{foreman-installer}` command on each {SmartProxy}:
```

Comment:
"each" -> "each relevant"? As "Some {SmartProxies} may be configured to use `pull-mqtt` mode while others use `ssh`."